x/superfluid/module.go

```diff
@@ -187,6 +187,7 @@ func (am AppModule) BeginBlock(ctx sdk.Context, _ abci.RequestBeginBlock) {
 // EndBlock executes all ABCI EndBlock logic respective to the capability module. It
 // returns no validator updates.
 func (am AppModule) EndBlock(ctx sdk.Context, _ abci.RequestEndBlock) []abci.ValidatorUpdate {
+	EndBlocker(ctx, am.keeper)
```

Comment:
What is the reason for having this `EndBlocker` function? Why can't we run that logic directly here?

Reply:
I don't know the exact reason behind this, but afaik, this is one of the conventions in settings endblockers, I'm guessing it's because we want to separate out logic for abci? 
cc @alexanderbez

Reply:
It would be ugly to have that defined here IMO. Having it encapsulated in a function makes it easier to test.

Actually, now that I think of it, this really should be defined on the keeper itself, just like `InitGenesis`. I.e. `(k Keeper) EndBlock(...)`